File: hackerrank/algorithms/01-warmup/01_solve-me-first/python/solveMeFirst.py
def solveMeFirst(a, b):
    # Check if a and b are within the valid range
    if 1 <= a <= 1000 and 1 <= b <= 1000:
        return a + b
    else:
        print("The numbers are not valid")
        return None
    
num1 = int(input())
num2 = int(input())
res = solveMeFirst(num1,num2)
print(res)

File: hackerrank/algorithms/01-warmup/01_solve-me-first/cpp/solveMeFirst.cpp
#include <iostream>
using namespace std;

int solveMeFirst(int a, int b) {
    // Check if a and b are within the valid range
    if (a >= 1 && a <= 1000 && b >= 1 && b <= 1000) {
        return a + b;
    } else {
        cout << "The numbers are not valid" << endl;
        return -1; // Return -1 to indicate invalid input
    }
}

int main() {
    int num1, num2;
    cin >> num1 >> num2;  // Input numbers
    int res = solveMeFirst(num1, num2);
    
    if (res != -1) { // Only print the result if it's valid
        cout << res << endl;
    }
    
    return 0;
}
